src/components/08-miscellaneous/07-popup/popup.njk

```diff
@@ -0,0 +1,213 @@
+                </div>
+                <div class="tablet:grid-col-auto sds-popup text-left">
+                  <i class="sds fa-information-circle fa-2x"></i>
+                  <div class="sds-popup__content in top-right margin-bottom-neg-05">
+                    <div class="radius-lg shadow-2 margin-y-neg-1x05 width-card-lg bg-white padding-left-1 padding-bottom-1 border">
+                      <i class="sds fa-information-circle fa-2x float-right margin-1"></i><h4> Pop inside</h4>
+                      <ul>
+                        <li>Lorem ipsum sit amet</li>
+                        <li>Lorem ipsum</li>
+                      </ul>
+                    </div>
+                    </div>
+                </div>
+              </div>
+              <div class="grid-row">
+                <div class="tablet:grid-col">
+                  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In neque nunc, auctor a cursus vitae,
+                    venenatis dapibus ex. Nam tristique aliquam efficitur. Aliquam sit amet sodales elit, accumsan
+                    lacinia
+                    nisi. Aliquam eu mauris ut augue fringilla congue a eu ante. Donec maximus placerat convallis. Morbi
+                    eu tincidunt neque, vel porta nulla. Suspendisse venenatis augue eget massa ullamcorper, et
+                    fringilla
+                    ipsum lobortis. In hac habitasse platea dictumst. Quisque et fringilla lacus, non consequat nulla.
+                  </p>
+                </div>
+              </div>
+              <div class="grid-row">
+                <div class="tablet:grid-col-auto sds-popup">
+                  <i class="sds fa-user-circle fa-2x"></i>
+                  <div class="sds-popup__content in bottom-left  ">
+                    <div class="sds-alert bg-base-lightest ">
+                      <div class="sds-alert-icon">
+                        <i class="fa fa-exclamation"></i>
+                      </div>
+                      <span>Review Required</span>
+                    </div>
+                  </div>
+                </div>
+                <div class="tablet:grid-col text-center">
+                  <div class="sds-popup">
+                    <i class="sds fa-new-note fa-2x"></i>
+                    <div class="sds-popup__content in bottom-center">
+                      <div class="radius-md shadow-2 margin-0 width-card-lg 
+                       text-left bg-accent-cool padding-1 border">
+
+                          <p>The First item</p><hr class="thin"/>
+                          <p>The Second item</p><hr class="thin"/>
+                          <p>Another item</p><hr class="thin"/>
+                          <p>One More item</p><hr class="thin"/>
+                          <p>The Last item</p>
+
+                      </div>
+                    </div>
+                  </div>
+                </div>
+                <div class="tablet:grid-col-auto text-right sds-popup">
```

Comment:
**For all popups** please refer to the guidelines for accessibility:
https://www.w3.org/TR/wai-aria-1.2/#tooltip
http://accessibility.athena-ict.com/aria/examples/tooltip.shtml


Main concern is that they don't appear on focus, missing role attribute, and missing aria-describeby.